package.json

```diff
@@ -25,6 +25,7 @@
   },
   "devDependencies": {
     "babelify": "^7.3.0",
+    "babel-cli": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
     "browserify": "^14.3.0",
     "uglify-js": "mishoo/UglifyJS2#harmony",
```

Comment:
This should have been fixed in `master`. There's `uglify-es@3` now, instead of the `mishoo/UglifyJS2#harmony` hack.